pipelines/backend_tests.go

```diff
@@ -2,21 +2,49 @@ package pipelines
 
 import (
 	"context"
-
 	"dagger.io/dagger"
+	"fmt"
 	"github.com/grafana/grafana-build/containers"
+	"log"
+	"strings"
 )
 
+var IntegrationDatabases = []string{"sqlite", "mysql", "postgres"}
+
+// list of strings, string
+func Contains(arr []string, v string) bool {
+	for _, str := range arr {
+		if str == v {
+			return true
+		}
+	}
+	return false
+}
+
 // GrafanabackendTests runs the Grafana backend test containers for short (unit) and integration tests.
 func GrafanaBackendTests(ctx context.Context, d *dagger.Client, args PipelineArgs) error {
+
+	db := args.Context.String("database")
+	log.Println("db", db)
```

Comment:
If we're going to log here we should be clearer with the intention, probably something like, log.Println("Using database", db)

But this is probably not the right time to log this particular thing. Users will always see `sqlite` even if they haven't selected `--integration`. Lower in the file we log the db name whenever `--integration` is true so I think we could just delete this line and leave the one further down.

Reply:
Sorry, this was an extra line of debugging I accidentally left in. I am shamed.